### ISSUE TITLE
Form formaction button size/location control

### ISSUE THREAD

**Issue by ToniJ123** [sqlpage/SQLPage#720]
What are you building with SQLPage ?

Large system

What is your problem ? A description of the problem, not the solution you are proposing.

What are you currently doing ? Since your solution is not implemented in SQLPage currently, what are you doing instead ?

Form's formaction button is different size than validate button and you cannot control validate button's size,
but you can formaction button's size via width parameter (1-12), but even with 1 as width, it doesn't go to the same
size as validate button. Validate button size depents on the text length in it? Tiny thing but it would make more 
sense to be able to control those buttons size to same, and also drop this formaction button to the same row line
as validate (and reset).

![formaction_button](https://github.com/user-attachments/assets/f502a360-6159-4b20-807e-a8d0a470ca2d)

"Tyhjennä" is formaction
"Päivitä" is validate.

Thanks!

**Comment by lovasoa** [comment]
Your form should probably look like this:

```sql
select 
    'form'                        as component,
    ''                            as validate;
select 
    'data' as name,
    'text' as type,
    'Data' as label,
    TRUE   as required;
select 
    'a'             as name,
    'submit'        as type,
    ''     as label,
    6             as width,
    '?x=0'         as formaction,
    'a'            as value;
select 
    'b'           as name,
    ''          as label,
    'submit'      as type,
    6             as width,
    '?x=1'        as formaction,
    'b'           as value;
```

Have a look at the form encoding example in https://sql-page.com/component.sql?component=form

**Comment by ToniJ123** [comment]
Yeah this was my bad as 1. didn't know that validate button was possible to make hidden when using `'' as validate`
2. you can make formaction button to do the same things as validate.

So from my side this can be closed. Thank you very much!
